src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -328,7 +328,7 @@ function MoneyRequestView({
     const receiptViolations =
         transactionViolations?.filter((violation) => receiptViolationNames.includes(violation.name)).map((violation) => ViolationsUtils.getViolationTranslation(violation, translate)) ?? [];
     const shouldShowNotesViolations = !isReceiptBeingScanned && canUseViolations && ReportUtils.isPaidGroupPolicy(report);
-    const shouldShowReceiptHeader = isReceiptAllowed && (shouldShowReceiptEmptyState || hasReceipt) && canUseViolations && ReportUtils.isPaidGroupPolicy(report);
```

Comment:
Is this intentional? why do we want to remove `canUseViolations`?

Reply:
Header was added coupled with notice violations as noted [here](https://github.com/Expensify/App/issues/43076#issuecomment-2149205108), hence this check was present.

Now we want to show the header for all cases, as noted [here](https://github.com/Expensify/App/issues/43076#issuecomment-2151119591). So I've removed the violations check

cc: @getusha

Reply:
That makes sense to me.